astropy/table/sorted_array.py

```diff
@@ -269,7 +278,7 @@ def replace_rows(self, row_map):
         self.data = self.data[keep_rows]
         self.row_index = np.array([row_map[x] for x in self.row_index[keep_rows]])
 
-    def items(self):
+    def items(self) -> list[tuple[str, list["Row"]]]:
```

Comment:
The `list[int]` is an approximation, it is really `list[np.int64]` (on Mac).

```suggestion
    def items(self) -> list[tuple[tuple[Any, ...], list[int]]]:
```